gateway-release/home/conf/gateway-site.xml

```diff
@@ -134,6 +134,13 @@ limitations under the License.
         <description>A duration (in seconds) beyond a token’s expiration to wait before evicting its state. This configuration only applies when server-managed token state is enabled either in gateway-site or at the topology level.</description>
     </property>
 
+    <!-- @since 2.1.0 application path aliases -->
+    <property>
```

Comment:
I'm wondering if it's possible/beneficial to support this at the topology level.

Reply:
I filed https://issues.apache.org/jira/browse/KNOX-2964 to address this improvement.